Makefile.murl

```diff
@@ -25,6 +25,7 @@ SOURCES= src/acvp.c \
          src/acvp_kdf135_x963.c \
          src/acvp_kdf135_tpm.c \
          src/acvp_kdf108.c \
+         src/acvp_kas_ecc.c \
```

Comment:
I think Makefile.win may have gotten missed? :)

Reply:
Also not sure if I am just missing it, but is acvp_kas_ecc included? It isn't showing up on my diff

Reply:
Yes, adding Makefile.win and acvp_kas_ecc.c(get fewer comments when omitted :)